scripts/cargo-test.sh

```diff
@@ -1,4 +1,14 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
+
+cleanup_handler() {
+  for c in $(docker ps -a --format '{{.ID}}') ; do
+    docker kill "$c" || true
```

Comment:
you might be able to use the label: io.mayastor.test.$name to kill only our containers. also do we need to remove the network too?

Reply:
Nice, I've overlooked that one (network is also labeled):

```json
            "Labels": {
                "io.mayastor.test.cargo-test": "true"
            }
```

Network is not mission critical as it will not break the test. But yes, it would be nicer to clean it up as well.

Reply:
I have so far discovered `cargo-test rest child_location_test lvs-pool-grpc mayastor_master composer node rest_backend` as potential values for the `$name`. Unfortunately docker doesn't have a partial/regex match support on labels. So either `jq` or list or changing the semantics:

```
            "Labels": {
                "io.mayastor.test": "cargo-test"
            }
```

Thus `docker ps -a --filter "label=io.mayastor.test"` would return all containers with label set to any value.

Reply:
@tiagolobocastro can you please take a look at b96eb81? I've replaced `label` with `label_prefix` and introduced a new "static" label `$label_prefix.name` with value of `name`. This will look like 

```
        "Labels": {
            "io.mayastor.test.name": "cargo-test"
        }
```